src/main/scala/cromwell/engine/backend/StdoutStderr.scala

```diff
@@ -2,4 +2,4 @@ package cromwell.engine.backend
 
 import cromwell.binding.values.WdlFile
 
-case class StdoutStderr(stdout: WdlFile, stderr: WdlFile)
+case class StdoutStderr(stdout: WdlFile, stderr: WdlFile, backendLogs: Option[Map[String, WdlFile]] = None)
```

Comment:
It might deserve a rename yes, I'm not sure to what though. Maybe `callLogs` or something ?